packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -472,7 +472,18 @@ export function injectIntoGlobalHook(globalObject: any): void {
     // Do the same for any already injected roots.
     // This is useful if ReactDOM has already been initialized.
     // https://github.com/facebook/react/issues/17626
-    hook.renderers.forEach((injected, id) => {
+    let renderers = hook.renderers;
```

Comment:
```suggestion
    let { renderers } = hook;
```

Reply:
I'm a big fan of destructuring assignment approach. The former code snippet appears kinda redundant atleast IMO.